tests/app.test.js

```diff
@@ -110,7 +110,7 @@ describe('Various ways to open and close InVEST models', () => {
       model_run_name: 'carbon',
       model_human_name: 'Carbon',
     };
-    remote.dialog.showOpenDialog.mockResolvedValue(mockDialogData);
+    ipcRenderer.invoke.mockResolvedValue(mockDialogData);
```

Comment:
Hmm this is interesting. I was wondering if it made sense to be a little more specific with the mock and mock the `electron.dialog.showOpenDialog` that is called in `main.js`. But none of this IPC actually works in tests does it, since the electron processes are not actually running.  

So this seems right to me as basically equivalent to the tests when we used `remote`. But also makes me wonder if it's worth trying to cover more of this IPC code with tests? It could be done with puppeteer, or I guess it could be done by mocking the implementations of `ipcRenderer` and `ipcMain` with actual `Event` objects. I'm not sure if the latter is really worth the effort.

Reply:
Yea, when re working the tests it took me awhile to realize that this was just going to be pretty empty shell stuff for mocking `ipc`.

I think I agree that it might be better to cover this using puppeteer. I can try to add a few tests and see how it goes.

Reply:
We decided to punt on this for now.